contrail-kubernetes-node/hooks/contrail_kubernetes_node_utils.py

```diff
@@ -24,6 +25,18 @@ def get_context():
     ctx["container_registry"] = config.get("docker-registry")
     ctx["contrail_version_tag"] = config.get("image-tag")
 
+    ctx["nested_mode"] = config.get("nested_mode")
+    if ctx["nested_mode"]:
+        # container need string value for nested_mode
+        ctx["nested_mode"] = "1"
+        # use KUBERNESTES_NESTED_VROUTER_VIP value
+        netsted_mode_dict = json.loads(config.get("nested_mode_config"))
+        if "KUBERNESTES_NESTED_VROUTER_VIP" in netsted_mode_dict:
+            ctx["nested_mode_config"] = {
+                "KUBERNESTES_NESTED_VROUTER_VIP": netsted_mode_dict["KUBERNESTES_NESTED_VROUTER_VIP"]}
+        if not ctx["nested_mode_config"]:
```

Comment:
maybe it's simpler to check its presence in template instead of creating empty objects?